R/hmde_assign_data.R

```diff
@@ -2,13 +2,12 @@
 #'
 #' @param model_template output from hmde_model
 #' @param data Input data tibble with columns including time, y_obs, obs_index, and additionally ind_id for multi-individual models
-#' @param step_size Step size for numerical integration.
 #' @param ... data-masking name-value pairs allowing specific input of elements
 #'
 #' @return updated named list with your data assigned to Stan model parameters
 #' @export
 
-hmde_assign_data <- function(model_template, data = NULL, step_size = NULL, ...){
+hmde_assign_data <- function(model_template, data = NULL,...){
```

Comment:
this branch adds a vignette, but there's also some change here to R functions and Stan files. are these intended?

Reply:
Yes, step-size is only used in the linear model anyway.